### ISSUE TITLE
get_all_params() error: some tree_learner_options keys missed

### ISSUE THREAD

**Issue by khrisanfov** [catboost/catboost#926]
catboost version: 16.0
Operating System: Ubuntu 18.04
GPU: 1080 Ti

get_all_params() cause the following error:

```
---------------------------------------------------------------------------

CatBoostError                             Traceback (most recent call last)

<ipython-input-9-04f1b25fb0a6> in <module>
----> 1 model.get_all_params()
      2 

/usr/local/lib/python3.6/dist-packages/catboost/core.py in get_all_params(self)
   2207         if not self.is_fitted():
   2208             raise CatBoostError("There is no trained model to use get_all_params(). Use fit() to train model. Then use this method.")
-> 2209         return self._object._get_plain_params()
   2210 
   2211 

_catboost.pyx in _catboost._CatBoost._get_plain_params()

_catboost.pyx in _catboost._CatBoost._get_plain_params()

CatBoostError: catboost/libs/options/plain_options_helper.cpp:730: some tree_learner_options keys missed
```

Code to reproduce https://github.com/khrisanfov/catboost_issues/blob/master/get_all_params.ipynb

**Comment by dbakshee** [comment]
Hi, thank you for reporting the issue and providing the case. 
It reproduces with recently released 0.16.1.

**Comment by annaveronika** [comment]
This is fixed now, will be out in our next minor release. Now you can build from source.
